src/modules/git_commit.rs

```diff
@@ -201,6 +204,42 @@ mod tests {
         repo_dir.close()
     }
 
+    #[test]
+    fn test_render_commit_hash_with_tag_disabled() -> io::Result<()> {
+        let repo_dir = fixture_repo(FixtureProvider::Git)?;
+
```

Comment:
The test needs to create a tag:
```suggestion
    create_command("git")?
            .args(&["tag", "v1", "-m", "Testing tags"])
            .current_dir(&repo_dir.path())
            .output()?;
```

Reply:
Sorry about missing this. I think the test `test_render_commit_hash_with_tag_enabled` also seems to miss this, so I add.